openpype/modules/deadline/plugins/publish/submit_blender_deadline.py

```diff
@@ -83,8 +88,9 @@ def get_job_info(self):
                 machine_list_key = "Blacklist"
             render_globals[machine_list_key] = machine_list
 
-        job_info.Priority = attr_values.get("priority")
-        job_info.ChunkSize = attr_values.get("chunkSize")
+        job_info.Comment = context.data.get("comment")
```

Comment:
Should the comment come from the instance?

There's this global Instance Per Comment plug-in that you can enable to allow custom comments per instance: https://github.com/pypeclub/OpenPype/blob/871bf194da2e7743363697032a984c41a2e54fc4/openpype/plugins/publish/collect_comment.py#L35

If not used, [this](https://github.com/ynput/OpenPype/blob/871bf194da2e7743363697032a984c41a2e54fc4/openpype/plugins/publish/collect_comment.py#L83-L104) should be transferring the `context.data["comment"]` to the instance already.

As such, by using instance here you basically make sure you also support the (optional per family) instance per comment plug-in.

Would that work?

Reply:
I don't know enough about this and I ~~blatantly copied~~ took inspiration from the Max submitter to implement the Blender submitter 😅
I'll try it out :)

Reply:
Might want to then push a similar fix to max if this works here

Reply:
It works :)

@moonyuet we should consider doing it for Max as well :)